include/seqan3/range/views/pairwise_combine.hpp

```diff
@@ -535,7 +535,7 @@ class pairwise_combine_view : public std::ranges::view_interface<pairwise_combin
     //!\cond
         requires (!std::same_as<remove_cvref_t<other_range_t>, pairwise_combine_view>) &&
                  std::ranges::viewable_range<other_range_t> &&  // Must come after self type check to avoid conflicts with the move constructor.
-                 std::constructible_from<underlying_range_type, ranges::ref_view<std::remove_reference_t<other_range_t>>>
+                 std::constructible_from<underlying_range_type, std::ranges::ref_view<std::remove_reference_t<other_range_t>>>
```

Comment:
Line length.
The line was a;ready too long before this PR, but maybe you can fix it now.